packages/web/tests/components/NavBar/NavProfileMenu.test.tsx

```diff
@@ -1,15 +1,44 @@
 import React from 'react';
-import { render, screen } from '../../testUtils/testTools';
+import { render, screen, act } from '../../testUtils/testTools';
 import { NavProfileMenu } from '../../../src/components/NavBar/NavProfileMenu';
+import fetchMock from 'fetch-mock-jest';
+
+const mockCurrentUser = {
+  id: 123,
+};
+
+const wait = () => new Promise<void>((resolve) => setTimeout(() => resolve(), 0));
 
 describe('NavLink Components', () => {
-  it('renders correctly', async () => {
+  beforeEach(async () => {
+    jest.clearAllMocks();
+    fetchMock.reset();
+  });
+
+  it('renders correctly and without logged in user', async () => {
+    fetchMock.get('/api/users/me', 401);
     render(<NavProfileMenu />);
 
+    await act(wait);
+
+    expect(screen.getByRole('img')).toBeVisible();
```

Comment:
According to John, this is the kind of thing we should be adding to End to End testing, not to this type of test 😬 
```suggestion
```